.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: java
+
+#
+# INSTALL: setup the software, i.e. build & install
+#
+install:
+  - mvn clean install -DskipTests --fail-at-end
```

Comment:
sometimes `-DskipTests` doesn't work properly (only surefire 2.3)
we use older version of it, so I guess this approach won't work. Please fix

Reply:
@chatelao will you do any changes/respond the comment above?

Reply:
which versions do you use?

Reply:
`which versions do you use?`
2.12.1
@chatelao have you seen #449 PR? it looks like you and @modax are working on the same things.